sockjs-protocol-0.3.3.py

```diff
@@ -819,21 +819,30 @@ def test_request_headers_cors(self):
                 headers={'Origin': 'test', 'Access-Control-Request-Method': 'POST', 'Access-Control-Request-Headers': 'a, b, c'})
         self.assertTrue(r.status == 204 or r.status == 200)
         self.verify_cors(r, 'test')
-        self.assertEqual(r['Access-Control-Allow-Headers'], 'a, b, c')
+        # A preflight request can benefit from returning all headers that it allows a client to send, since these
+        # headers will be stored in the preflight cache (similar to how the request methods are stored) which can 
+        # potentially save a round trip for subsequent request that send other/more headers.
+        self.assertEqual(r['Access-Control-Allow-Headers'].strip(), 'a,b,c')
```

Comment:
Why the change in spacing? I'm not sure this will pass because `strip` removes leading and trailing whitespace, not within the string.